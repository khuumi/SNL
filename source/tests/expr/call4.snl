do func to (do func2), 16.9, (do func3 to 1.6, c), local a, 7