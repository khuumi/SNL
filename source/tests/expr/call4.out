Call func [(Call func2 []), (Float 16.9), (Call func3 [(Float 1.6), (Id Global c)]), (Id Local a), (Int 7)]